docs/guide/web/default-theme.md

```diff
@@ -114,6 +114,19 @@ In this case the default Mandelbrot stylesheet link will be output between the t
 This option **is not used** for applying styles to your _components_ - for information on how to include component stylesheets see the docs on linking to [static assets](../web/#static-assets).
 :::
 
+### highlightStyles
+Works similarly to the previous `styles` option, but only affects code blocks highlighted by [highlight.js](https://highlightjs.org/).
+
+```js
+{
+    "highlightStyles": "https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.0.0/styles/atom-one-dark.min.css"
```

Comment:
It's a detail but we link an example stylesheet from Highlight.js version 10 while still being on 9, aren't we?

Reply:
Good point, I've updated the line to use styles from the same version we use for highlighting.